.babelrc

```diff
@@ -1,6 +1,13 @@
 {
-  "presets": ["es2015", "stage-0"],
-  "plugins": [
-    "add-module-exports"
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
+          "node": "12",
```

Comment:
Node 10 hasn't reached its EOL, so why shouldn't we keep it?

![image](https://user-images.githubusercontent.com/3382565/98205245-bba7c800-1f72-11eb-9b13-82377129750d.png)

Reply:
ok I'll change to node 10